fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransactionType.java

```diff
@@ -59,7 +59,7 @@ public enum LoanTransactionType {
     CHARGE_REFUND(24, "loanTransactionType.chargeRefund"), //
     CHARGEBACK(25, "loanTransactionType.chargeback"), //
     CHARGE_ADJUSTMENT(26, "loanTransactionType.chargeAdjustment"), //
-    CHARGE_OFF(27, "loanTransactionType.chargeOff");
+    CHARGE_OFF(27, "loanTransactionType.chargeOff"), DOWN_PAYMENT(28, "loanTransactionType.downPayment");
```

Comment:
Please follow the code styling of the LoanTransactionType enum: New entries goes to new line

Reply:
done